themes/default/content/docs/intro/console/auditing.md

```diff
@@ -56,8 +61,8 @@ query parameter must be passed and is used to query audit records _before_ the s
 GET https://api.pulumi.com/api/orgs/${org}/auditlogs/export?startTime=${time}
 ```
 
-Optionally, a user to filter by can also be specified as a query parameter to filter audit logs
-pertaining only to a specifc user.
+Optionally, a user can also be specified as a query parameter to filter audit logs
+pertaining only to a specific user.
```

Comment:
maybe something like "Query parameters can also be used to filter records involving a specific user" or something else a little tighter?